xml/manager_modules.xml

```diff
@@ -64,7 +64,7 @@
    <para>
     To view the current balancer configuration, run:
    </para>
-<screen>&prompt.cephuser;ceph config-key dump</screen>
+<screen>&prompt.cephuser;ceph balancer status</screen>
```

Comment:
It seems misleading to say "here's how to get the current balancer _configuration_", and then give a command that displays the balancer's _status_.

Would it make more sense to write here:

```
Current Balancer Status

To view the current balancer status, run:

$ ceph balancer status
```

?

Reply:
But it does show the configuration as well, mode and plan are config values, right?

Reply:
ok, how about "Current Balancer Status and Basic Configuration" ?

Reply:
That makes sense, I adjusted accordingly, but we now have a new PR, sorry about that.